core/templates/components/shared-component.module.ts

```diff
@@ -30,17 +30,21 @@ import { SharingLinksComponent } from
   './common-layout-directives/common-elements/sharing-links.component';
 import { ExplorationEmbedButtonModalComponent } from
   './button-directives/exploration-embed-button-modal.component';
+import { SocialButtonsComponent } from
+  'components/button-directives/social-buttons.component';
 import { NgbModalModule } from '@ng-bootstrap/ng-bootstrap';
 
 
+
 @NgModule({
   imports: [CommonModule, NgbModalModule],
   declarations: [BackgroundBannerComponent, TranslatePipe,
     SharingLinksComponent, ExplorationEmbedButtonModalComponent,
-    SkillMasteryViewerComponent],
+    SkillMasteryViewerComponent, SocialButtonsComponent],
   entryComponents: [BackgroundBannerComponent, SharingLinksComponent,
     // These elements will remain here even after migration.
-    ExplorationEmbedButtonModalComponent, SkillMasteryViewerComponent
+    ExplorationEmbedButtonModalComponent, SkillMasteryViewerComponent,
+    SocialButtonsComponent
```

Comment:
The skill mastery and social buttons will go up, above the comment on line 45.

Reply:
@orthodoxparadox do not forget to resolve this comment in next PR.